### PR TITLE
feat(context functions): expose core functions for ease of use

### PR DESCRIPTION
**Pull request by coreylight**
By exposing functions like mark, metric, and label at the module level,
consumers of this library will have an easier time using those features.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,10 +35,32 @@ exports.handler = iopipe((event, context) => {
 });
 ```
 
-By default this package will enable the tracing plugin. For more information on how to use Iopipe and the tracing plugin, see the documentation below:
+By default this package will enable @iopipe/trace and @iopipe/event-info plugins. For more information on how to use IOpipe and these plugins, see the documentation below:
 - [IOpipe Documentation](https://github.com/iopipe/iopipe-js-core#readme)
 - [IOpipe Tracing Plugin Documentation](https://github.com/iopipe/iopipe-plugin-trace#readme)
 
+Example With Tracing, Custom Metrics, and Labels (ES6 Module Format):
```
Comment (pselle) [inline README.md]: 💯

**Comment by coreylight** [comment]
@pselle @kolanos I realized I left `.log` in here, I'm thinking it should be removed as it is deprecated?

**Comment by pselle** [comment]
@coreylight ah, didn't catch that. Yes, I think you could remove that from the exports

**Comment by coreylight** [comment]
@pselle @kolanos back in review - removed `.log` export.

**Pull request merged**